### PR TITLE
DM-31163: Enable section for flags from forced_src

### PR DESCRIPTION
**Pull request by yalsayyad**
in the Object Table specification. Specify with forcedFlags header

### CODE REVIEW

File: python/lsst/pipe/tasks/functors.py
Comment (arunkannawadi) [inline python/lsst/pipe/tasks/functors.py:554]: This should preferably occur after the block involving `flags`. That way, if a flag appears twice by mistaken, then the forced_src will be the one that in the object table, assuming that's what we'd want.

File: python/lsst/pipe/tasks/postprocess.py
Comment (arunkannawadi) [inline python/lsst/pipe/tasks/postprocess.py:458]: The `forcedFlags` argument is not documented yet. Similarly, in line 605 it should be mentioned that `forcedFlags` will also be expanded per band.

**Review by arunkannawadi — APPROVED** [review]

**Comment by arunkannawadi** [comment]
Approving it for now so as to not block the merge before tonight, but changes are requested.

File: python/lsst/pipe/tasks/functors.py
Comment (yalsayyad) [reply python/lsst/pipe/tasks/functors.py:554]: ...and I do want the regular meas flags to be the default.

**Comment by yalsayyad** [comment]
The "changes requested" button is usually for big refactoring changes that require a second review because the rework is new code.

**Pull request merged**
